File: share-code/bai-tap-lap-trinh/cpp/31.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{
    int start       = 1;
    int end         = 30;
    string result;
    string notice   = "Không có";
    //  tìm giá trị chia hết cho 13 trong khoảng 1 đến 30
    //  13 26 
    // 2,3,4...29
    for (int i = start + 1; i < end; i++)
    {
        if (i % 13 == 0)
        {
            result += to_string(i) + "\n";
        }  
    }

    if (result.length() != 0)
    {
        notice = result;
    }

    cout << notice;
}

File: share-code/bai-tap-lap-trinh/python/loop/31.py
start = 39
end = 40
result = ""

for i in range(start - (start % 13) + (13 if start % 13 != 0 else 0), end, 13):
    result += str(i) + " "

result = result.rstrip() if result else "Không có"

print(result)